book/pipelines.md

```diff
@@ -138,3 +140,35 @@ drwxr-xr-x@  4 pengs  admin   128 14 Nov 13:42 en
 ```nu
 > ls /usr/share/nvim/runtime/ | get name | find tutor | ^ls -al $in
 ```
+
+## Command Output in Nushell
+
+Unlike external commands, Nushell commands are akin to functions. Most Nushell commands do not print anything to `stdout` and instead just return data.
+
+```nu
+> do { ls; ls; ls; "What?!" }
+```
+
+This means that the above code will not display the files under the current directory three times.
+In fact, running this in the shell will only display `"What?!"` because that is the value returned by the `do` command in this example. However, using the system `^ls` command instead of `ls` would indeed print the directory thrice because `^ls` does print its result once it runs.
+
+Knowing when data is displayed is important when using configuration variables that affect the display output of commands such as `table`.
+
+```nu
+> do { $env.config.table.mode = none; ls }
+```
+
+For instance, the above example sets the `$env.config.table.mode` configuration variable to `none`, which causes the `table` command to render data without additional borders. However, as it was shown earlier, the command is effectively equivalent to
+
+```nu
+> do { $env.config.table.mode = none; ls } | table
```

Comment:
should we add the working example as well for completion?
```nushell
do { $env.config.table.mode = none; ls | table }
```

Reply:
Sure